### DISCUSSION TITLE
neue IP-Adresse nach Wartung des Senec-Speichers

### DISCUSSION THREAD

**Discussion by stocki1972** [solectrus/solectrus#3571] [Q&A] [answered]
### Beschreibung (auf Deutsch oder Englisch)

Hallo,
nach einer Wartung des Senec-Speichers durch einen Fachbetrieb hat dieser plötzlich eine neue IP-Adresse .200.
Wie kann ich diese neue Adresse Solectrus mitteilen, damit diese wieder eine Verbindung zum Speicher aufnehmen kann?
Bitte für Dummies Schritt-für-Schritt erklären.
Vielen Dank
Martin

### Welche Version von SOLECTRUS verwendest du?

v017.01

### Auf welchem Gerät läuft bei dir SOLECTRUS?

Raspberry Pi 4

### Dein vollständiges `docker-compose.yml` oder `compose.yml`

```yaml
Keine Ahnung
```

### Dein vollständiges `.env`

```text
Keine Ahnung
```

**Comment by ledermann** [comment]
Die IP-Adresse des SENEC-Speichers steht in der Datei `.env` in der Variablen `SENEC_HOST`, siehe Doku hier:
https://docs.solectrus.de/referenz/senec-collector/konfiguration.html#senec_host

Du kannst das einfach ändern:

1) `.env` editieren und IP-Adresse anpassen
2) Neu starten mit `docker compose up -d`

**Comment by stocki1972** [comment]
Was muss ich da für Befehle eingeben? Wie gesagt, ich hab null Ahnung davon

**Reply by hambue** [reply, marked as answer]
Öffne den Ordner mit den Konfigurationsdateien:
`cd solectrus`

Öffne die Datei mit den Umgebunsvariablen:
`nano .env`

Dort zur Zeile mit der IP-Adresse scrollen und die letzten Ziffern ändern. Dann mit `Ctrl+X` speichern und mit `y` bestätigen.

Zum Schluss: `docker compose up -d`

**Reply by stocki1972** [reply]
Super. Habs geschafft!!!! DANKE vielmals :-)

**Comment by Zeppelin-PV** [comment]
Ich habe dieses Problem über den Router gelöst.
Dort lässt sich einstellen dass ein Teilnehmer immer die gleiche IP Adresse
zugewiesen bekommt. Voraussetzung dafür ist, dass sich der Speicher über DHCP eine IP Adresse abholt und diese nicht statisch vergeben wurde.
